boards/README.md

```diff
@@ -51,7 +51,7 @@ In general, to prepare the Vitis platform, users have the following options:
 * Rebuild Vitis platform from source
 (e.g. [source for Xilinx official platforms](https://github.com/Xilinx/Vitis_Embedded_Platform_Source/tree/master/Xilinx_Official_Platforms))
 * Download an official Vitis platform
-(e.g. [ZCU104 platform](https://www.xilinx.com/bin/public/openDownload?filename=zcu104_dpu_2019.2.zip))
+(e.g. [ZCU104 `platform`](https://www.xilinx.com/bin/public/openDownload?filename=zcu104_dpu_2020.1.zip))
```

Comment:
This is not valid anymore. Maybe point to the official zcu104 platform here: https://www.xilinx.com/support/download/index.html/content/xilinx/en/downloadNav/embedded-platforms/2020-1.html